lib/hijack/db.js

```diff
@@ -164,7 +164,14 @@ hijackDBOps = function hijackDBOps() {
           }
         } else if(type == 'fetch' || type == 'map'){
           //for other cursor operation
+
           endData.docsFetched = ret.length;
+
+          var coll = cursorDescription.collectionName;
+          var query = cursorDescription.selector;
+          var opts = cursorDescription.options;
+          var docSize = Kadira.docSzCache.getSize(coll, query, opts, ret);
+          endData.docSize = docSize * ret.length;
```

Comment:
Here we need to track these and cursorDocSize. Which should go to either the method or publication depending on the trace type.